src/components/Account/AccountModule.js

```diff
@@ -1,145 +1,81 @@
 import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react'
 import { useWallet } from 'use-wallet'
-import { Button, GU, IconConnect, springs, useViewport } from '@aragon/ui'
-import { Transition, animated } from 'react-spring/renderprops'
-import { shortenAddress, getUseWalletProviders } from '../../lib/web3-utils'
+import { Button, GU, IconConnect, useViewport } from '@aragon/ui'
+import { shortenAddress } from '../../lib/web3-utils'
 import AccountButton from './AccountButton'
-import HeaderPopover from '../Header/HeaderPopover'
+import AccountPopover from './AccountPopover'
 import ScreenConnected from './ScreenConnected'
 import ScreenConnecting from './ScreenConnecting'
 import ScreenError from './ScreenError'
 import ScreenProviders from './ScreenProviders'
 
 const SCREENS = [
-  {
-    id: 'providers',
-    title: 'Use account from',
-    height:
-      4 * GU + // header
-      (12 + 1.5) * GU * Math.ceil(getUseWalletProviders().length / 2) + // buttons
-      7 * GU, // footer
-  },
-  {
-    id: 'connecting',
-    title: 'Use account from',
-    height: 38 * GU,
-  },
-  {
-    id: 'connected',
-    title: 'Active account',
-    height: 22 * GU,
-  },
-  {
-    id: 'error',
-    title: 'Connection error',
-    height: 50 * GU,
-  },
+  { id: 'providers', title: 'Use account from' },
+  { id: 'connecting', title: 'Use account from' },
+  { id: 'connected', title: 'Active account' },
+  { id: 'error', title: 'Connection error' },
 ]
 
 function AccountModule() {
-  const buttonRef = useRef()
-  const wallet = useWallet()
   const [opened, setOpened] = useState(false)
-  const [animate, setAnimate] = useState(false)
   const [activatingDelayed, setActivatingDelayed] = useState(false)
```

Comment:
```suggestion
  const [activatingDelayed, setActivatingDelayed] = useState(null)
```

Reply:
Done in https://github.com/aragon/court-dashboard/pull/375/commits/3c757b38a0b01dd3fdfebffd763ad3170faff9f2